src/ngx-gallery-scroll-overview/ngx-gallery-scroll-overview.component.scss

```diff
@@ -0,0 +1,13 @@
+.image-box {
+    background-size: contain;
+    background-image:
+        url("http://blog.doctoroz.com/wp-content/uploads/2013/07/AP_07_30_2013_53726226_L.jpg");
```

Comment:
Here you should take the image that is used in the gallery. Maybe the smaller image that is also used in the gallery? (I know we don't use this at the moment, but as soon as we provide a smaller image for the ngx-gallery (see https://www.npmjs.com/package/ngx-gallery#ngxgalleryimage) this component should also use it)

To make it dynamic you have to set the css attribute `background-image` on the image box via `ngStyle` (see https://stackoverflow.com/a/34875479/5816097)